DEV_5/DEV_5/Bird.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace DEV_5
+{
+    using System;
+
+    /// <inheritdoc />
+    /// <summary>
+    /// The bird.
+    /// </summary>
+    public class Bird : IFlyable
+    {
+        /// <summary>
+        /// Birds current speed.
```

Comment:
it is not needed to use summaries for all fields. In some cases it enough to use valid Name